.circleci/set-up-globals.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
```

Comment:
A lot of this file is duplication/slight variation of [`/build-tools-ci/scripts/set-environment` in `docker-build-tools-ci`](https://github.com/pantheon-systems/docker-build-tools-ci/blob/4.x/scripts/set-environment) and [`.circleci/set-up-globals.sh` from `example-wordpress-composer`](https://github.com/pantheon-systems/example-wordpress-composer/blob/a0187e98cd5283dd580c775621553149929769bf/.circleci/set-up-globals.sh).

Reply:
See my comment: https://github.com/pantheon-systems/example-wordpress-composer/pull/85/files#r269383419

It would be better to adapt the `set-environment` script to be useful for multiple providers. We could pass in `CIRCLE_BUILD_NUMBER` et. al. as parameters, and it could set generic `TERMINUS_` and `CI_` variables for us.

This might necessitate a 5.x version of the build container, unless perhaps we make it backwards-compatible (read `CIRCLE_` variables if nothing passed in).